polar-core/benches/bench.rs

```diff
@@ -191,6 +191,16 @@ pub fn many_rules(c: &mut Criterion) {
     });
 }
 
+fn parser(c: &mut Criterion) {
+    let policy = include_str!("roles_policy.polar");
+    c.bench_function("parse_policy", |b| {
+        b.iter(|| {
+            let polar = Polar::new();
+            polar.load_str(policy).unwrap()
```

Comment:
This is a lot more than just the parser. Would benching running `parse_lines` on the string be a better benchmark?